community/event-funding.tt

```diff
@@ -0,0 +1,105 @@
+[% WRAPPER layout.tt title="Event funding" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html">Community</a>
+    <span>→</span>
+  </div>
+  <h1>Event funding</h1>
+</div>
+
+<section class="community-event-funding">
+  <p>
+    The NixOS community is awesome! You guys have been generously donating
+    throughout the year to keep the NixOS projects running smoothly. We want to
+    show our appreciation by launching a new grant program that will help our
+    community thrive even more.
+  </p>
+  <p>
+    This program will be aimed at supporting and boosting the NixOS community,
+    which has been instrumental in getting us where we are today. We hope that
+    it will continue to reward and encourage amazing folks like you to
+    volunteer your time and talent for years to come.
+  </p>
+
+  <div class="community-event-funding-button">
+    <a href="https://commotion.page/view/event-funding">Apply</a>
+  </div>
+
+  <h2>Who can apply?</h2>
+  <ul>
+    <li>
+      Any active community member that’s looking to organize an event with Nix
+      developers, publicly announced.
+    </li>
+    <li>
+      Recipient must be in a SEPA country or in a country supported by Wise
```

Comment:
```suggestion
      The recipient must be located in a SEPA country or in a country supported by Wise. This allows to avoid money transfer issues. Unfortunately, individuals from India are excluded: it proved to be difficult to deal with India in terms of transactions. However, if you have a company, you can apply: things are easier this way.
```

Reply:
```suggestion
      The recipient must be located in a SEPA country or in a country supported by Wise.
      This allows to avoid money transfer issues. Unfortunately, individuals from India are excluded:
      it proved to be difficult to deal with India in terms of transactions.
      However, you can apply if you have a company, as things are easier this way.
```